FlowCal/excel_ui.py

```diff
@@ -605,13 +734,32 @@ def add_beads_stats(beads_table, beads_samples):
         FCSData objects from which to calculate statistics.
         ``beads_samples[i]`` should correspond to
         ``beads_table.values()[i]``.
+    mef_outputs : list, optional
+        A list with the intermediate results of the generation of the MEF
+        transformation functions, as given by ``mef.get_transform_fxn()``.
+        If specified, ``mef_outputs[i]`` should correspond to
+        ``beads_table.values()[i]``.
```

Comment:
might be nice to specify which columns depend on the `mef_outputs` parameters (i.e. which columns will be excluded if you don't provide `mef_outputs`)